portfolio/src/main/webapp/script.js

```diff
@@ -154,12 +154,47 @@ const selectionChangeHandler = async (value) => {
   loadComments(value);
 }
 
+/**
+ * Get a specific GET parameter from the query string
+ * @param{String} name The name of the parameter
+ */
+const getParameter = (name) => {
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  return urlParams.get(name);
+}
+
+/**
+ * Set the value of the #maxComments element on the page
+ * @param{Number} value The value to be assigned
+ */
+const setMaxComments = (value) => {
+  const maxCommentsElement = document.getElementById('maxComments');
+  const selectedIndex = maxCommentsElement.selectedIndex;
+  const optionValues = [];
+  // constructing the optionValues[] array in this way is required as
```

Comment:
I would put the comment above optionValues since that is part the code you are explaining

Reply:
fixed 👍 https://github.com/thomasogara/step-repository/pull/27/commits/7389f7020e377c9ab3600a40b57a37064759690e